packages/ketcher-react/src/script/editor/tool/rotate-controller.ts

```diff
@@ -1,5 +1,10 @@
 import { Action, Scale, Vec2 } from 'ketcher-core'
 import { throttle } from 'lodash'
+import {
+  hideFloatingTools,
+  showFloatingTools,
+  updateFloatingToolsPosition
+} from 'src/script/ui/state/floatingTools'
```

Comment:
just noticed, if possible we should avoid coupling the `Editor` and the UI redux event.

So we have mapped in the `packages/ketcher-react/src/script/ui/state/editor/index.js` and in the current implementation you have: 
```
onUpdateFloatingTools: memoizedDebounce((action) => {
  dispatch(action)
})
```
but probably we should do smth like that so, real mapping:
```
onUpdateFloatingTools: memoizedDebounce((params: UpdateFloatingToolsData) => {
if(params ....)
  dispatch(hideFloatingTools())
else ...
   dispatch(showFloatingTools(...))
else ...
 dispatch(updateFloatingToolsPosition(...))
})
```

In this case, UI will know about the interface of the Event of `Editor` (`UpdateFloatingToolsInterface`), but `Editor` will not know about UI logic.